tests/factories.py

```diff
@@ -198,7 +199,7 @@ class Meta:
     creator = SubFactory(AuthUserFactory)
 
 
-class ProjectFactory(AbstractNodeFactory):
+class ProjectFactory(NodeFactory):
```

Comment:
I changed this because I was getting an error about saving a `AbstractNode` without the type field being set.